wombase_backend/server/apps/authentication/views.py

```diff
@@ -1,3 +1,37 @@
-from django.shortcuts import render
+from django.contrib.auth import authenticate
+from rest_framework import status
+from rest_framework.authtoken.models import Token
+from rest_framework.generics import CreateAPIView
+from rest_framework.response import Response
 
-# Create your views here.
+from .serializers import EmployeeRegistrationSerializer, EmployeeLoginSerializer
+
+
+class EmployeeRegistrationView(CreateAPIView):
+    serializer_class = EmployeeRegistrationSerializer
+    permission_classes = ()
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+        user = serializer.save()
+
+        return Response(serializer.data)
+
+
+class EmployeeLoginView(CreateAPIView):
+    serializer_class = EmployeeLoginSerializer
+    permission_classes = ()
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+
+        phone_number = serializer.initial_data.get('phone_number')
+        password = serializer.initial_data.get('password')
```

Comment:
why initial data?

Reply:
If we use serializer.data, it will throw AssertionError. You have to validate data using serializer.is_valid() or use initial_data